package.json

```diff
@@ -50,6 +49,7 @@
     "@shelf/jest-mongodb": "^1.2.3",
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.0.2",
+    "brotli-webpack-plugin": "^1.1.0",
```

Comment:
Might need to checkout this webpack plug-in!

Reply:
yeah. this helped me with text compression.